src/components/logger/index.ts

```diff
@@ -26,22 +26,15 @@ import chalk from 'chalk'
 import { AxiosResponseWithExtraData } from '../../interfaces/request'
 import { Probe } from '../../interfaces/probe'
 import { Notification } from '../../interfaces/notification'
-import { saveProbeRequestLog, getAllLogs, saveNotificationLog } from './history'
+import { getAllLogs, saveProbeRequestLog } from './history'
 import { log } from '../../utils/pino'
 
-import { Alerts, LogObject } from '../../interfaces/logs'
+import { LogObject } from '../../interfaces/logs'
+import { getEventEmitter } from '../../utils/events'
+import { PROBE_LOGS_BUILT } from '../../constants/event-emitter'
+import { saveNotificationLog } from '../logger/history'
 
-// declare monika logs and initialize
-let mLogs = {
-  alert: {
-    flag: '',
-    message: [],
-  } as Alerts,
-  notification: {
-    flag: '',
-    message: [],
-  } as Alerts,
-} as LogObject
+const em = getEventEmitter()
```

Comment:
Can we find better variable name for this `em`?
It's hard for me to guess `em` as `EventEmitter` if I dont read this line.

Reply:
agreed!